### DISCUSSION TITLE
How am i setup lavalink server

### DISCUSSION THREAD

**Discussion by Kai-Developer** [SudhanPlayz/Discord-MusicBot#563] [Q&A] [answered]
Hello, I don't know how to install lavalink, can someone who knows about this help me?

**Comment by lombed** [comment]
it is also interesting to learn how to connect in docker-compose

**Comment by JotaroKujo0525** [comment]
Ask in the discord server not here

**Comment by DarrenOfficial** [comment, marked as answer]
Hello there, Heres how you can install lavalink
On Windows & Mac: https://youtu.be/aLfI4ameT9o
On Replit: https://youtu.be/bznq-P0Ovzk

Hope this helps!
